src/config/keys.ts

```diff
@@ -139,5 +143,7 @@ export const MUTATION_KEYS = {
   POST_ITEM_LIKE: 'postItemLike',
   DELETE_ITEM_LIKE: 'deleteItemLike',
   ADD_FAVORITE_ITEM: 'addFavoriteItem',
-  DELETE_FAVORITE_ITEM: 'deleteFavoriteItem'
+  DELETE_FAVORITE_ITEM: 'deleteFavoriteItem',
+  POST_VALIDATION: 'postItemValidation',
+  UPDATE_VALIDATION_REVIEW: 'updateValidationReview'
```

Comment:
I think you should stick with one convention. Always add `item` so: 
```
POST_ITEM_VALIDATION: 'postItemValidation',
  UPDATE_ITEM_VALIDATION_REVIEW: 'updateitemValidationReview'

```

Maybe check all your code respect this convention.